src/ui/roi_selection_dialog.py

```
@@ -0,0 +1,551 @@
+        # Selection preview
+        if self._selection_mode and self._polygon_points:
+            self._preview_pos = QPointF(x, y)
+            self._update_overlay()
+
+    def _handle_mouse_release(self, event: QMouseEvent, scene_pos: QPointF) -> None:
+        if event.button() == Qt.LeftButton and self._dragging_handle != ROIHandle.NONE:
+            self._dragging_handle = ROIHandle.NONE
+            self._last_drag_pos = None
+            self._update_overlay()
+
+    def _handle_mouse_double_click(self, event: QMouseEvent, scene_pos: QPointF) -> None:
```

Ruff (0.14.14)

[warning] Unused method argument: `scene_pos`

(ARG002)